docs/transparent-validation/sd-wire/faq.md

```diff
@@ -0,0 +1,35 @@
+# SDWire Frequently asked questions (FAQ)
+
+## Will the SDWire be compatible with the Odroid-XU4?
+
+The SDWire has been tested on the RTE (Remote Testing Environment) platform with
+Orange Pi Zero and GoodRam MicroSDHC 32 GB Class 10 UHS-I/U1.
+
+For usage documentation, please visit SDWire [usage validation](https://docs.dasharo.com/transparent-validation/sd-wire/usage-validation/) 
+section.
+
+Please note that the "First use" has already been completed during the pre-sale
+validation of the SDWire. However, we haven't conducted compatibility tests on
+the Odroid-XU4, so we cannot provide specific details regarding compatibility
+with those devices. You can learn more about the RTE platform
+[here](https://shop.3mdeb.com/shop/open-source-hardware/rte/).
+
+## Can I use this product to remotely download photos from a DSLR camera? I'm concerned about speed.
+
+Yes, the product is designed to facilitate remote access and data management for
+SD cards, which should work well for your purpose of retrieving photos from a
+remote DSLR camera.
+
+For additional details please refer to the official documentation:
+
+* [Dasharo Universe](https://docs.dasharo.com/transparent-validation/sd-wire/getting-started/)
+* [Tizen Wiki](https://wiki.tizen.org/SDWire)
+
+These resources will provide you with comprehensive guidance on how to make the
+most of the product for your specific needs.
+
+## What are the speed limitations, specifically the maximum speed for reading and writing to an SD card?
+
+We conducted a simple test involving multiple attempts to flash an SD card
+inserted into SD Wire. The data transfer rates for these successive attempts
+ranged between 12 MB/s and 21 MB/s.
```

Comment:
We should be clear about what hardware we used to test. We can link to the compatibility section.

Reply:
fixed in the newest commit